src/models/transfers.js

```diff
@@ -461,10 +453,26 @@ function * getFulfillment (transferId) {
   return fulfillment.getDataExternal()
 }
 
+function * resendNotification (transferId) {
```

Comment:
Not sure that's necessary, but that's one option. To be clear, we are doing it via ws for performance, right? Otherwise, RC could check for transfers that have not been finalized and use the `get transfer` endpoint to get the state of those transfers from the ledger.